### ISSUE TITLE
Restrict Editing Permissions

### ISSUE THREAD

**Issue by tpcolson** [nationalparkservice/places-editor#75]
We'd like to see some sort of editing permission workflow where either ONLY pre-approved park employees are allowed to edit park authoritative data in Places, and/or any edits are approved by the park before being made public, include a review of any required FGDC metadata which accompanies all park GIS data made available to the public. We do NOT want volunteers or other non-qualified editors making changes, or adding to, authoritative data.

**Comment by nateirwin** [comment]
Yep, this is on our radar. @jimmyrocks and I had a conversation about the workflow this morning, and we'll likely start implementing in the next few weeks.

In the meantime, we are working on a change-within implementation that will allow you to subscribe to notifications for your park. This should be a good interim solution until the validation workflow is ready to go.

**Comment by bhousel** [comment]
Hey!  I have a work in progress for defining editing restrictions in iD that will do a lot of this for you. I'll try to finish it up this week and merge into master....

https://github.com/openstreetmap/iD/commit/8467433418f944f71069d3817bc2f09503f9e862

**Comment by jimmyrocks** [comment]
@bhousel, this sounds like something we could definitely put to use in our application. We want to have a tag that signifies if something is validated. Any user can change the object, but the validate tag won't get copied over to a new version unless an authorized user specifies it.

**Comment by nateirwin** [comment]
:+1:

**Comment by ethomas33** [comment]
Along this line could we add a "notes" options so someone could go in drop a point and say I don't agree with this and why but they can't actually edit anything?

**Comment by jimmyrocks** [comment]
I like this idea! Right now we're just using github issues to track this
stuff, but OSM has some notes support. So we can look into mimicking that.

**Comment by tpcolson** [comment]
..or some variation where a unit has to approve new/edited data before it is made available via the CartoDB extract.
